schemas/user.yml

```diff
@@ -54,6 +54,10 @@
         as: real_user
         short_override: The real user (ruid). Identifies the real owner of the process.
         beta: Reusing the `user` fields in this location is currently considered beta.
+      - at: process
+        as: attested_user
+        short_override: The attested user (auid). Identifies the attested used associated with the process.
```

Comment:
Also, i know there isn't a ton of space for descriptions on nested fields. "short only", but I wonder if we can explain what attested means. e.g via 2factor mechanisms etc...

Reply:
"externally" attested is the key aspect ie they have done something to prove to an external source (e.g. pw & 2nd factor) that they are who claim to be and authorized to obtain the role/identity stated here.